### ISSUE TITLE
Aggregators - Merge Plugin Configuration

### ISSUE THREAD

**Issue by mohsin106** [influxdata/telegraf#6483]
I'm trying to use the merge aggregator with Telegraf v1.12 Alpine Docker container, but getting following error message:

2019-10-04T14:42:48Z E! [telegraf] Error running agent: Error parsing /etc/telegraf/telegraf.conf, Undefined but requested aggregator: merge

I didn't see any documentaiton on how to use the merge plugin so I'm guessing its not fully integrated yet?

**Comment by Theoooooo** [comment]
There is a bit of documentation about it 
https://github.com/influxdata/telegraf/tree/master/plugins/aggregators/merge

**Comment by mohsin106** [comment]
Under the configuration section it shows:
[[aggregators.merge]]
  # no configuration

How do I pass the plugin fields that I want to be aggregated into one metric? 

I'm currently receiving a router interface name and its parent-ae-name in separate data sets. First the interface name comes in and then the parent-ae-name. I want to aggregate these two fields into one field. Would this plugin do that?

**Comment by Theoooooo** [comment]
This mean that there is no more configuration to do for this plugin to work like the system plugin 
```
# Read metrics about system load & uptime
[[inputs.system]]
  # no configuration
```

**Comment by danielnelson** [comment]
This plugin will be included in the 1.13 release, or you can test it out with the [nightly builds](https://github.com/influxdata/telegraf#nightly-builds).
